src/utils/changelog.ts

```diff
@@ -252,6 +253,28 @@ interface ReleaseConfig {
   };
 }
 
+/**
+ * Normalized release config with Sets for efficient lookups
+ */
+interface NormalizedReleaseConfig {
+  changelog: {
+    exclude?: {
+      labels?: Set<string>;
+      authors?: Set<string>;
+    };
+    categories?: NormalizedCategory[];
```

Comment:
Don't make anything optional in normalized configs. Empty sets and arrays are easier.